### ISSUE TITLE
Cocina mapping error: DescriptiveBasicValue does not define properties: script

### ISSUE THREAD

**Issue by jcoyne** [sul-dlss/dor-services-app#1157]
```
druid:bc943qv3518 error. #/components/schemas/DescriptiveBasicValue does not define properties: script
```

Caused by:
```ruby
      Cocina::Models::Description.new(
        :title=>[{:value=>"batch2 virtual child 473"}],
        :adminMetadata=>{
          :language=>[
            {
              :value=>"English",
              :code=>"eng",
              :uri=>"http://id.loc.gov/vocabulary/iso639-2/eng",
              :source=>{:code=>"iso639-2b", :uri=>"http://id.loc.gov/vocabulary/iso639-2"},
              :script=>{:value=>"Latin", :code=>"Latn", :source=>{:code=>"iso15924"}}
            }
          ],
        }
      )
```

**Comment by ndushay** [comment]
if the code for recordInfo re-uses the mappings for the top level elements, then <scriptTerm> has already been addressed by #1256.

**Comment by jcoyne** [comment]
@ndushay this was fixed by upgrading to cocina-models 0.42.1
